packages/flutter/lib/src/widgets/animated_scroll_view.dart

```diff
@@ -663,6 +877,20 @@ typedef AnimatedItemBuilder = Widget Function(BuildContext context, int index, A
 ///   instead of removing them.
 typedef AnimatedRemovedItemBuilder = Widget Function(BuildContext context, Animation<double> animation);
 
+/// Signature for the builder callback used in [AnimatedList.separated]
+/// to animate its separators after they have been removed.
+///
+/// The [context] argument is the build context where the widget will be
+/// created, the [index] is the index of the corresponding item of the separator
+/// that is removed, and the [animation] is an [Animation] that should be used to
+/// animate an exit transition for the widget that is built.
+///
+/// See also:
+///
+/// * [AnimatedItemBuilder], a builder that is for adding items with animations
+///   instead of removing them.
+typedef AnimatedRemovedSeparatorBuilder = Widget Function(BuildContext context, int index, Animation<double> animation);
```

Comment:
Same here

Reply:
I tried to update the `AnimatedItemBuilder` docstring to describe its usage for the removedSeparatorBuilder. But I'm unsure, if I was able to describe it in an easy to understand way. If we should improve it further, do you have ideas in what way?